evalml/tests/component_tests/test_imputer.py

```diff
@@ -334,34 +335,72 @@ def test_imputer_multitype_with_one_bool(data_type, make_data_type):
     imputer = Imputer()
     imputer.fit(X_multi, y)
     X_multi_t = imputer.transform(X_multi)
-    assert_frame_equal(X_multi_expected_arr, X_multi_t.to_dataframe())
+    assert_frame_equal(X_multi_expected_arr, X_multi_t)
+
+
+def test_imputer_int_preserved():
+    X = pd.DataFrame(pd.Series([1, 2, 11, np.nan]))
+    imputer = Imputer(numeric_impute_strategy="mean")
+    transformed = imputer.fit_transform(X)
+    pd.testing.assert_frame_equal(transformed, pd.DataFrame(pd.Series([1, 2, 11, 14 / 3])))
```

Comment:
Why the integer division here?

Reply:
I'm not sure I follow, `14 / 3` does float division.

Reply:
Oh... sorry yes you're right.

In python 2, `14 / 3` produces `4`. In python 3, it produces `4.333333`. Lol.